### PR TITLE
adding VSTS tutorial to index

### PR DESCRIPTION
**Pull request by barlanmsft**
Iain's module

### CODE REVIEW

File: docs/deploy-azure/toc.yml [hunk 1]
```diff
@@ -32,6 +32,8 @@
     href: /vsts/build-release/actions/define-multistage-release-process?toc=/vsts/deploy-azure/toc.json&bc=/vsts/deploy-azure/breadcrumb/toc.json
   - name: CI builds for Git in VSTS
     href: /vsts/build-release/actions/ci-build-git?toc=/vsts/deploy-azure/toc.json&bc=/vsts/deploy-azure/breadcrumb/toc.json
+  - name: Deploy apps to Azure with VSTS and CI/CD
```
Comment (iainfoulds) [inline docs/deploy-azure/toc.yml]: I'd suggest Deploy Web Apps to Azure with VSTS and CI/CD as the name, since it is very much about Web Apps.

**Comment by steved0x** [comment]
@barlanmsft Is this PR ready to merge or are you still iterating on review feedback?

**Comment by barlanmsft** [comment]
@steved0x this PR is ready for merge #sign-off

**Comment by barlanmsft** [comment]
Actually, @steved0x - this needs to merge _after_ we set our tutorials to go live for Build. Can we #hold-off for now and we confirm again once we're ready to get this published?

**Comment by erickson-doug** [comment]
@barlanmsft is this ready to live? @yunyunyunyun said the corresponding content has been posted.

**Comment by erickson-doug** [comment]
the learn content is live: https://docs.microsoft.com/learn/deploy-apps-to-azure/index  -- am merging this so the vendor cross-repo merge will pick this up.

**Pull request merged**